mpo.rga.Dayon.yml

```
@@ -0,0 +1,61 @@
+app-id: mpo.rga.Dayon
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: /app/dayon_assistant.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=home:rw
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: dayon
+    buildsystem: simple
+    build-commands:
+      - tar -xf dayon.tgz
+      - install dayon/dayon.jar /app
+      - install dayon/dayon_*.sh /app
+      - install dayon.sh /app
```

```suggestion
      - install -Dm755 dayon.sh -t /app/bin
```

any executable should be in `/app/bin`. Or symlinked there. Definitely NOT in `/app`